mintpy/ifgram_inversion.py

```diff
@@ -1070,6 +1088,7 @@ def ifgram_inversion(ifgram_file='ifgramStack.h5', inps=None):
     # Parallel loop
     else:
         try:
+            import dask
```

Comment:
If you move the config_name fallback logic to `create_parser` this import statement can be removed.

Reply:
We need the fallback logic after read_template2inps().

Reply:
This import can still be removed, because `dask` will have already been imported before this code every runs.

Reply:
I did not realize it, you are right.

Reply:
Now I remembered. `dask` has to be imported here because the one imported in cmd_line_parse() is a local variable that won't be existed within the ifgram_inversion().